graphistry/feature_utils.py

```diff
@@ -907,6 +907,7 @@ def process_dirty_dataframes(
         try:
             X_enc = data_encoder.fit_transform(ndf, y)
         except TypeError:
+            logger.info("obj columns:", object_columns, "are being converted to str")
```

Comment:
@dcolinmorgan `logger.info("obj columns: %s are being converted to str", object_columns)`

Reply:
+ ci failing?

Reply:
message was placed before object_columns was defined